content/github/authenticating-to-github/creating-a-personal-access-token.md

```diff
@@ -53,15 +53,15 @@ Personal access tokens (PATs) are an alternative to using passwords for authenti
    **Warning:** Treat your tokens like passwords and keep them secret. When working with the API, use tokens as environment variables instead of hardcoding them into your programs.
 
    {% endwarning %}
-{% if currentVersion == "free-pro-team@latest" %}9. To use your token to authenticate to an organization that uses SAML SSO, [authorize the token for use with a SAML single-sign-on organization](/articles/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on).{% endif %}
+{% if currentVersion == "free-pro-team@latest" %}9. To use your token to authenticate to an organization that uses SAML SSO, [authorize the token for use with a SAML single-sign-on organization](/github/authenticating-to-github/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on).{% endif %}
```

Comment:
Weirdly, this link still isn't rendering:

![image](https://user-images.githubusercontent.com/42641846/114151586-29d32b00-9915-11eb-83a7-65f58140a797.png)

Your syntax is correct, so it looks like a bug with the rendering! Slightly adjusting whitespace should help—I've pushed a fix!